docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,592 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        discord_idx = np.argmax(
</pre><p>maybe slightly cleaner to do:</p><pre class="ql-syntax" spellcheck="false">discord_idx = np.where(np.isinf(PL), np.NINF, PL).argmax()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>